hooks/beforeRegistration.ts

```diff
@@ -1,35 +1,16 @@
 import { currentStoreView } from '@vue-storefront/core/lib/multistore'
 
 export function beforeRegistration({ Vue, config, store, isServer }) {
-  const VSF_PAYPAL_CODE = 'vsfpaypal'
-  store.dispatch('payment/addMethod', {
-    'title': 'Paypal',
-    'code': VSF_PAYPAL_CODE,
-    'cost': 0,
-    'costInclTax': 0,
-    'default': false,
-    'offline': true
-  })
+  // const VSF_PAYPAL_CODE = 'vsfpaypal'
+  const VSF_PAYPAL_CODE = 'paypal_express'
 
-  if (!Vue.prototype.$isServer) {
+  if (!isServer) {
     const storeView = currentStoreView()
     const { currencyCode } = storeView.i18n
-    const clientId = config.paypal.clientId
-    const sdkUrl = `https://www.paypal.com/sdk/js?client-id=${clientId}&currency=${currencyCode}`
+    const clientId = config.paymentPaypalMagento2.clientId
+    const sdkUrl = `https://www.paypal.com/sdk/js?client-id=${clientId}&currency=${currencyCode}&disable-funding=card,credit&intent=authorize`
```

Comment:
@tonyisworking Need move the load paypal sdk to the mount level of button compoent.